runtime/src/lib.rs

```diff
@@ -369,19 +370,21 @@ impl pallet_timestamp::Config for Runtime {
     type WeightInfo = weights::pallet_timestamp::ZKVWeight<Runtime>;
 }
 
-/// Existential deposit.
-pub const EXISTENTIAL_DEPOSIT: u128 = MILLICENTS;
+parameter_types! {
+    /// Existential deposit.
+    pub const ExistentialDeposit: Balance = EXISTENTIAL_DEPOSIT;
+}
 
 impl pallet_balances::Config for Runtime {
     type MaxLocks = ConstU32<50>;
-    type MaxReserves = ();
+    type MaxReserves = ConstU32<50>;
```

Comment:
Just curiosity ... why?

Reply:
I was using this to test some `xcm` scenarios. Anyways, I will add some e2e tests to cover the basic xcm interactions that we want to support, and then remove this if unneeded.